BazyDanych/BazaDanych/BazaDanych/Model/Author.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace BazaDanych.Model
+{
+    internal class Author
+    {
+        public Author(string name, string surname)
+        {
+            Name = name;
+            Surname = surname;
+        }
+
+        public Author()
+        {
+        }
+
+        public int AuthorId { get; set; }
```

Comment:
SUG: Dobrze by było zeby modele nie były modyfikowalne:
|
```suggestion
        public int AuthorId { get; }
```

Reply:
[4a920e2](https://github.com/broniuu/Praktyki2022/pull/3/commits/4a920e2aa946ade78630174a4048fbc69600a0b7)